### PR TITLE
Search bar - Connecting back-end to front-end (#69)

### PR DESCRIPTION
**Pull request by Ryan2601**
Connection is done.

Some changes had to be made in the (getVoters.js) query files. it appears that using % charachter which is need for the LIKE statement in sql makes error once running. So the solution for that is to add the % charachters in the passed arguments rather than writing it in the sql statment itself.

screenshot from 2018-09-17 16-06-08

Please notice - PR #67 should be merged first.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ class App extends Component {
   };
 
   render() {
+    console.log("render in app.js");
```
Comment (Karyum) [inline client/src/App.js]: remove this log, you shouldn't have console logs in the frontend

File: client/src/components/boxViewContainer.js [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
-    box: "",
+    box: '',
     voters: [],
-    votersAmount: "",
-    voted: ""
+    votersAmount: '',
+    voted: ''
   };
   searchVoter = e => {
+    console.log('search voer in BOX VIEW');
```
Comment (Karyum) [inline client/src/components/boxViewContainer.js]: :up:

File: server/routes/index.js [hunk 3]
```diff
@@ -1,12 +1,21 @@
 const express = require('express');
 const path = require('path');
+const getVotersFromDB = require('../helpers/getVotersHelper');
 
 const router = express.Router();
-const voters = require('../model/voters');
+// const voters = require('../model/voters');
```
Comment (Karyum) [inline server/routes/index.js]: why is this commented out ?

Comment (tawfiknasser) [reply server/routes/index.js]: we are using this for tests( the connection between the back end and the front end), replacing the database.

**Comment by tawfiknasser** [comment]
thanks you @Karyum  :wave:

**Pull request merged**